guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -11,6 +11,15 @@ There are no highlights with Katello {KatelloVersion}.
 
 There are no upgrade warnings with Katello {KatelloVersion}.
 
+=== Port 8443 disabled by default
+
+On {SmartProxyServer}, the deprecated port 8443 has been disabled by default on new installations and upgrades.
+Clients are expected to be using port 443 when communicating through a {SmartProxyServer}.
+If there are older clients still using port 8443, they should be re-configured either by using remote execution, the katello-ca-consumer RPM, or updating `/etc/rhsm/rhsm.conf` using `subscription-manager --port 443`.
```

Comment:
Do we have a specific documented procedure for this that we could link to? While the release notes aren't directly used in downstream, a procedure would and AFAIK this will also be relevant to downstream customers.

Reply:
We could link to the host management guide, perhaps a sub-section but there is no direct procedure for this.

Reply:
Should we have that? If so, who would be responsible for writing it?

Reply:
This can vary by use-case, and what feature sets are used by a given customer. And https://community.theforeman.org/t/rfc-replace-pub-katello-rhsm-consumer-static-script-with-a-template/38494/21 is standardizing this to an endpoint for configuration. I think the documentation that comes with that change should suffice.